cache.py

```diff
@@ -8,6 +8,12 @@ def purge(*args):
     for path in args:
         run("curl -s -I -X PURGE http://localhost:7999%s | grep '200 Purged'" % path.strip())
 
+@task
+@roles('class-cache', 'class-draft_cache')
+def purge_all():
+    "Purge everything from varnish. This is done by restarting all varnish processes."
+    restart()
```

Comment:
I agree. I have a memory that @rjw1 has done it before?

I suppose if we think it is risky enough, we could stagger the restarts? There are currently 3 cache machines, plus 2 draft cache machines.